modules/azure/disk.go

```diff
@@ -0,0 +1,101 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// DiskExists indicates whether the specified Azure Managed Disk exists.
+// This function would fail the test if there is an error.
+func DiskExists(t testing.TestingT, diskName string, resGroupName string, subscriptionID string) bool {
```

Comment:
Was everything in this file supposed to be part of https://github.com/gruntwork-io/terratest/pull/618?

Reply:
Yes, they were introduced at different times but are valuable independently.

Reply:
Note this test does use the GetDiskType which was described as a thin wrapper in the Disk PR.  It adds more code to the test example but updated this VM test to check for the property via the disk object.